src/main/app/src/app/zaken/zaak-create/zaak-create.component.ts

```diff
@@ -14,7 +14,7 @@ import { NavigationService } from "../../shared/navigation/navigation.service";
 import { UtilService } from "../../core/service/util.service";
 import { Vertrouwelijkheidaanduiding } from "../../informatie-objecten/model/vertrouwelijkheidaanduiding.enum";
 import { AbstractFormField } from "../../shared/material-form-builder/model/abstract-form-field";
-import { Observable, Subject } from "rxjs";
+import {Observable, of, Subject} from "rxjs";
```

Comment:
Formatting issue? Our .editorconfig does not guard against this (removal of spaces)?

Reply:
for some reason my formatter is not working in intellij (I just switched from vscode because it's unworkable with java) will fix it